### PR TITLE
Added game log to end game screen

### PR DESCRIPTION
**Pull request by gomfunkel**
Added the game log to the end game screen as requested in issue #255.

### CODE REVIEW

File: src/styles/game_end.less [hunk 1]
```diff
@@ -24,3 +24,22 @@
         border-bottom: .05rem solid #7b7b7b !important;
     }
 }
+
+.game_end_block--log {
```
Comment (alrusdi) [inline src/styles/game_end.less]: This style definition duplicates code from player_home.less
Please, reuse it, do not copy/paste

Comment (gomfunkel) [reply src/styles/game_end.less]: I absolutely see your point but as all classnames are prefixed with the screen they are on I figured this is the preferred way you do it. If it's not: How do you handle reuse of styles in general, I can't image adding a `.player_home_block--log` class to the game end screen would be the preferred way, is it?

Comment (alrusdi) [reply src/styles/game_end.less]: The best variant - is to move log styling to separate log.less file, and make its css-classes naming independent from page name.
Simplest variant is just modify player_home.less and add .game_end_block--log to .player_home_block--log class

Comment (gomfunkel) [reply src/styles/game_end.less]: @alrusdi Fixed.

**Pull request merged**